R/chart.R

```diff
@@ -0,0 +1,87 @@
+#' Chart the application of a function along coordinates
+#'
+#' \code{chart()} modifies a function that takes a list to make it act
```

Comment:
I can understand all the words individually but I don't really get what you're trying to say :/